CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [1.38.109](https://github.com/brave/brave-browser/releases/tag/v1.38.109)
+
+ - Added Gemini two-way Brave Rewards wallet. ([#17408](https://github.com/brave/brave-browser/issues/17408))
+ - Added the ability to redirect an AMP page to its canonical non-AMP version. ([#21643](https://github.com/brave/brave-browser/issues/21643))
+ - Added the ability to copy Brave Wallet transaction hash from "Transactions" on the "Portfolio" page. ([#19886](https://github.com/brave/brave-browser/issues/19886))
```

Comment:
that one is desktop related

Reply:
Nice catch, not sure how it was added as it doesn't include `OS/Android`. Removing 👍